testtweetme.py

```diff
@@ -0,0 +1,89 @@
+import os
```

Comment:
Can we make a test directory and place tests separately in different files?

Reply:
Sure. let me take care of it.